go/api/bitmap_commands.go

```diff
@@ -0,0 +1,99 @@
+// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+import "github.com/valkey-io/valkey-glide/go/glide/api/options"
+
+// Supports commands and transactions for the "Bitmap" group of commands for standalone and cluster clients.
+//
+// See [valkey.io] for details.
+//
+// [valkey.io]: https://valkey.io/commands/#bitmap
+type BitmapCommands interface {
+	// Sets or clears the bit at offset in the string value stored at key.
+	// The offset is a zero-based index, with 0 being the first element of
+	// the list, 1 being the next element, and so on. The offset must be
+	// less than 2^32 and greater than or equal to 0 If a key is
```

Comment:
```suggestion
	// The offset is a zero-based index, with `0` being the first element of
	// the list, `1` being the next element, and so on. The offset must be
	// less than `2^32` and greater than or equal to `0` If a key is
```
Consider copying docs from node client